be/src/olap/hll.h

```diff
@@ -154,7 +174,7 @@ class HyperLogLog {
     HllDataType _type = HLL_DATA_EMPTY;
 
     uint32_t _explicit_data_num = 0;
-    uint64_t _explicit_data[HLL_EXPLICIT_INT64_NUM * 2];
+    uint64_t* _explicit_data = nullptr;
```

Comment:
directly use the raw ptr is dangerous and complex. Rethink shoule we replace is use std::unique_ptr

Reply:
no need replace with unique_ptr,  keep style consistency with uint8_t *_registers